content/spin/install.md

```diff
@@ -90,7 +90,7 @@ To install the canary version of spin, you should pass the argument `-v canary`.
 
 {{ startTab "Windows"}}
 
-On Windows systems you can download <a href="https://github.com/fermyon/spin/releases/tag/v0.9.0" class="spin-install" id="spin-install-windows">the Windows binary release of Spin</a>.
+On Windows systems you can download <a href="https://github.com/fermyon/spin/releases/tag/v0.10.0" class="spin-install" id="spin-install-windows">the Windows binary release of Spin</a>.
```

Comment:
Have we considered changing this to `latest` and Never Speaking Of It Again?

Reply:
`latest` it is i.e. `https://github.com/fermyon/spin/releases/latest` works like a charm.